apps/site/assets/css/_whats-happening.scss

```diff
@@ -1,6 +1,21 @@
 .m-whats-happening__section {
   margin-bottom: $base-spacing;
   width: 100%;
+
+  h2 {
```

Comment:
Sorry, can we remove this section's styling so that this heading uses the same sizing as the rest of the `<h2>` on this page?